### ISSUE TITLE
Build errs out of box: "!contains PS5 def" and warns

### ISSUE THREAD

**Issue by dylanh724** [PlayFab/PlayFabMultiplayerUnreal#29]
Place PlayFab dir in \Plugins >> Generate .uproj files >>

> While compiling \Intermediate\Build\BuildRules\FooProjectModuleRules.dll:

> \Plugins\Online\PlayFabMultiplayerUnreal\Source\OnlineSubsystemPlayFab.Build.cs(21,98) : error CS0117: **'UnrealBuildTool.UnrealTargetPlatform' does not contain a definition for 'PS5'**

> \Plugins\Online\PlayFabMultiplayerUnreal\Source\OnlineSubsystemPlayFab.Build.cs(170,53) : error CS0117: **'UnrealBuildTool.UnrealTargetPlatform' does not contain a definition for 'PS5'**

I don't want to even support PS5 - I tried editing `\Plugins\PlayFab\PlayFab.uplugin`, but there's not even PS5 listed; only:

> [ "Win64", "Win32", "Mac", "IOS", "Android", "PS4", "XboxOne", "Switch" ]

or maybe the error is because it's _not_ included here and expected (I'll try this soon)?

_______________

In [the quickstart](https://learn.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/party-unreal-engine-oss-quickstart), they said I should edit the target builds that I don't want in the uplugin -- so now it's just:

```json
{
	"Name": "OnlineSubsystemPlayFab",
	"Enabled": true,
	"WhitelistPlatforms": [
		"Win64"
	],
	"SupportedTargetPlatforms": [
		"Win64"
	]
}
```

...which results in warnings:

> WARNING: Unknown platform XboxOneGDK listed in plugin with FriendlyName Online Subsystem PlayFab Party & Multiplayer
> WARNING: Unknown platform WinGDK listed in plugin with FriendlyName Online Subsystem PlayFab Party & Multiplayer
> WARNING: Unknown platform XSX listed in plugin with FriendlyName Online Subsystem PlayFab Party & Multiplayer
> WARNING: Unknown platform PS5 listed in plugin with FriendlyName Online Subsystem PlayFab Party & Multiplayer
> 
> \Plugins\Online\PlayFabMultiplayerUnreal\Source\OnlineSubsystemPlayFab.Build.cs(21,98) : error CS0117: 'UnrealBuildTool.UnrealTargetPlatform' does not contain a definition for 'PS5'
> \Plugins\Online\PlayFabMultiplayerUnreal\Source\OnlineSubsystemPlayFab.Build.cs(170,53) : error CS0117: 'UnrealBuildTool.UnrealTargetPlatform' does not contain a definition for 'PS5'

**Comment by nassosterz-ms** [comment]
We have logged a bug and will fix this, thanks for reporting it!

**Comment by nassosterz-ms** [comment]
We are closing this issue as this has been the design that we have followed so far. This error does not affect anything in the functionality. We might consider dealing with it in a future update of the OnlineSubsystemPlayFab plugin.

**Comment by dylanh724** [comment]
@nassosterz-ms As the name of the ticket implied, there were build errors out of the box - not just warnings. I believe you may have just seen the warnings above. Please see the bottom of the logs that also included errors. Ps5 doesn't exist as a target platform.

I saw there's a way to implicitly find build targets that you guys actually partially used. This is the way it should be done to prevent errors outside the box.

**Comment by nassosterz-ms** [comment]
Let me reopen this issue as we are still considering it. Thanks!

**Comment by yuehanlin** [comment]
This issue is fixed since release 2.3.0 https://learn.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/unreal-release-notes#230, so I am closing this issue. Please feel free to reopen if you have any questions!
